setup.py

```diff
@@ -497,71 +495,63 @@ def srcpath(name=None, suffix=".pyx", subdir="src"):
     "_libs.tslib": {
         "pyxfile": "_libs/tslib",
         "depends": tseries_depends,
-        "sources": ["pandas/_libs/tslibs/src/datetime/np_datetime.c"],
     },
     "_libs.tslibs.base": {"pyxfile": "_libs/tslibs/base"},
     "_libs.tslibs.ccalendar": {"pyxfile": "_libs/tslibs/ccalendar"},
     "_libs.tslibs.dtypes": {"pyxfile": "_libs/tslibs/dtypes"},
     "_libs.tslibs.conversion": {
         "pyxfile": "_libs/tslibs/conversion",
         "depends": tseries_depends,
-        "sources": ["pandas/_libs/tslibs/src/datetime/np_datetime.c"],
+        # TODO: xstrtod symbol visibility is really murky here
+        "include": klib_include,
+        "sources": ["pandas/_libs/src/parser/tokenizer.c"],
```

Comment:
Removing the np_datetime.c source yielded complaints from the linker about not being able to find `xstrtod`, which is defined in `tokenizer.c`; a more Python approach may be to make tokenizer a capsule in another PR